sonar-python-plugin/src/test/java/org/sonar/plugins/python/pylint/PylintRulesDefinitionTest.java

```diff
@@ -43,7 +43,7 @@ public void pylint_external_repository() {
     RulesDefinition.Rule rule = repository.rule("C0121");
     assertThat(rule).isNotNull();
     assertThat(rule.name()).isEqualTo("Singleton comparison");
-    assertThat(rule.htmlDescription()).isEqualTo("This is external rule <code>pylint:C0121</code>. No details are available.");
+    assertThat(rule.htmlDescription()).isEqualTo("Used when an expression is compared to singleton values like True, False or None.");
```

Comment:
I'm not sure whether we want that: is that what we did for other analyzers? Should we keep those descriptions up-to-date with newer versions of pylint?